libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -99,6 +99,16 @@ - (void)viewDidLayoutSubviews {
 
 - (void)viewWillAppear:(BOOL)animated {
     [super viewWillAppear:animated];
+    UIWindow *currentWindow = (UIWindow *) self.view.window;
```

Comment:
Should we move that block of code to some helper method (maybe in the SFSDKUIWindow class) so that one would just need a one line call?

Reply:
@wmathurin Its a good suggestion,  should we just add it to a base UIViewController 
 SDK instead?

Reply:
Are you thinking of swizzling in a new implementation of viewWillAppear for UIViewController that would refresh the views or to simply add a helper method to the class via a category?

Reply:
I was thinking its easier for devs to call super instead if class is extended.